web/packages/teleport/src/Account/ManageDevices/AddAuthDeviceWizard/AddAuthDeviceWizard.tsx

```diff
@@ -306,7 +453,25 @@ function SavePasskeyStep({
               onChange={onNameChange}
               readonly={saveAttempt.attempt.status === 'processing'}
             />
-            <ButtonPrimary type="submit">Save the Passkey</ButtonPrimary>
+
+            {usage === 'mfa' && newMfaDeviceType === 'otp' && (
+              <FieldInput
+                label="Authenticator Code"
+                labelTip="Enter the code generated by your authenticator app"
+                rule={requiredField('Authenticator code is required')}
+                inputMode="numeric"
+                autoComplete="one-time-code"
+                value={authCode}
+                placeholder="123 456"
+                onChange={onAuthCodeChanged}
+                readonly={saveAttempt.attempt.status === 'processing'}
+              />
+            )}
+            <ButtonPrimary type="submit">
+              {usage === 'passwordless'
+                ? 'Save the Passkey'
+                : 'Save the MFA method'}
```

Comment:
Can the button just say "Save" ??

The title of the dialog box already says "Save the MFA method" so I don't see why we need to repeat that much test twice.

Reply:
@zmb3 That was what we settled down on with the UX team. @rishibarbhaya-design, could you confirm?

Reply:
For what it's worth, when you edit an auth connector or a role, the save button says "SAVE CHANGES" - it does not say "Save the role" or "Save the auth connector"

Reply:
I'm okay with "Save".